pkg/chart/loader/testdata/frobnitz.v2.reqs/Chart.lock

```diff
@@ -0,0 +1,8 @@
+dependencies:
```

Comment:
If there are dependencies in the lock file shouldn't the appropriate piece be in the `Chart.yaml` file, too?

Reply:
That could only be described as a human error mistake! :) Apologies, will remove.

Reply:
If you add the dependencies to the `Chart.yaml` I'll come through and approve this one given the other conversation is moving to a follow-up.

Reply:
The lock file should not have been there as the dependencies are in the requirements. The test is to show that v3 will handle chart specified as apiVersion "v2" but using dependencies in requirements file.